content/source/docs/enterprise/admin/logging.html.md

```diff
@@ -1,86 +1,303 @@
 ---
 layout: "enterprise"
-page_title: "Logging - Infrastructure Administration - Terraform Enterprise"
+page_title: "Log Forwarding - Infrastructure Administration - Terraform Enterprise"
 ---
 
-# Terraform Enterprise Logs
+# Terraform Enterprise Log Forwarding
 
-This document contains information about interacting with Terraform Enterprise logs. There are two types of logs, application logs and audit logs. Application logs emit information about the services that comprise Terraform Enterprise. Audit logs emit information whenever any resource managed by Terraform Enterprise is changed.
+Terraform Enterprise supports forwarding its logs to one or more external
+destinations; aptly referred to as log forwarding. This page details how to
+configure log forwarding in Terraform Enterprise.
 
-## Application Logs
+## Requirements
 
-Terraform Enterprise runs in a set of Docker containers. As such, any tooling that can interact with Docker logs can read the logs. This includes the command `docker logs`, as well as access via the [Docker API](https://docs.docker.com/engine/api/v1.36/#operation/ContainerLogs).
+Log forwarding requires:
 
-An example of a tool that can automatically pull logs for all docker containers is [logspout](https://github.com/gliderlabs/logspout).
-Logspout can be configured to take the Docker logs and send them to a syslog endpoint. Here's an example invocation:
+- Terraform Enterprise running on an instance with the `systemd-journald`
+service started and enabled. Execute `systemctl status systemd-journald` to
+check if the `systemd-journald` service is started and enabled.
+- A version of Docker that supports the `journald` logging driver. Execute
+`docker info --format '{{.Plugins.Log}}'` to check if the `journald` plugin is
+listed.
+- Network connectivity between Terraform Enterprise and the desired external
```

Comment:
Should we mention the default port here?

Reply:
No need for this. All the connections are egress from Terraform Enterprise to the desired destination. The ports for the destinations can vary.